nix/default.nix

```diff
@@ -285,10 +285,36 @@ let
         cp -r generated-docs $out
       '';
 
+  launchSearchablePursDocs = { builtDocs ? null, ... }:
```

Comment:
There's still no info on how to invoke it from the shell (`nix build` or `nix run` line with arguments)

Reply:
What do you mean by arguments? Also everything else in this module is undocumented, might as well document them now too